Symfony/CS/Tests/FixerTest.php

```diff
@@ -209,6 +209,9 @@ public function getFixersPriorityCases()
             array($fixers['short_bool_cast'], $fixers['spaces_cast']),
             array($fixers['unneeded_control_parentheses'], $fixers['trailing_spaces']), // tested also in: trailing_spaces,unneeded_control_parentheses.test
             array($fixers['class_definition'], $fixers['trailing_spaces']), // tested also in: class_definition,trailing_spaces.test
+            array($fixers['empty_return'], $fixers['useless_return']), // tested also in: empty_return,useless_return,whitespacy_lines,extra_empty_lines.test
+            array($fixers['useless_return'], $fixers['whitespacy_lines']), // tested also in: empty_return,useless_return,whitespacy_lines,extra_empty_lines.test
+            array($fixers['useless_return'], $fixers['extra_empty_lines']), // tested also in: empty_return,useless_return,whitespacy_lines,extra_empty_lines.test
```

Comment:
Please, also add integration test for only pair of fixers - one could run only 2 of them.